### PR TITLE
RLintBear.py: Add asciinema link

### PR DESCRIPTION
**Pull request by khashf**
### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

### CODE REVIEW

**Comment by khashf** [comment]
@refeed: PR title and commit message has been changed via commit https://github.com/coala/coala-bears/pull/2381/commits/543e4af9de33077321bc80c24833438b7b323615. Could you please review again?

File: bears/r/RLintBear.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class RLintBear:
                     DistributionRequirement(apt_get='r-base',
                                             version='>=3.1.1')}
     LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/FIq6VmrcdruXdgdGxwG9DdyuD'
```
Comment (jayvdb) [inline bears/r/RLintBear.py]: asciinema is not public

Comment (khashf) [reply bears/r/RLintBear.py]: @jayvdb I've updated the link. Please review.

**Comment by khashf** [comment]
@jayvdb @newbazz @li-boxuan Link has been made public.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Squash your commits.  See http://coala.io/git

**Comment by jayvdb** [comment]
ack 9cf340f

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
reack https://github.com/coala/coala-bears/pull/2381/commits/c8390d0a2d02a4d1e60259aba3397a49f5a07664

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**